packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -4288,9 +4303,16 @@ export default class Meeting extends StatelessWebexPlugin {
       joinSuccess = resolve;
     });
 
+    if (options.correlationId) {
+      this.setCorrelationId(options.correlationId);
+      LoggerProxy.logger.log(
+        `Meeting:index#join --> Using a new correlation from app ${this.correlationId}`
```

Comment:
very minor thing, but to be consistent with other logs it would be better to say "correlation id" instead of just "correlation"

Reply:
fixed.